src/guide/making-a-change.md

```diff
@@ -68,6 +68,12 @@ If you create a new file, copy the license text from a nearby file. Do not add c
 
 If you move code from an existing file to a new file, bring the entire copyright + license header over to the new file.
 
+## Changes Late in the Release
+
+You should always aim to get your changes integrated as early in a release as possible to maximize the amount of testing done before release. If you risk running late it's almost always a better idea to defer the change to the next release rather than trying to squeeze it in in the last minute. The next release is only six months out, the world can wait even though you are eager.
```

Comment:
```suggestion
You should always aim to get your changes integrated as early in a release as possible to maximize the amount of testing done before release. If you risk running late, it's almost always a better idea to defer the change to the next release rather than trying to squeeze it in in the last minute. The next release is only six months away, the world can wait even though you are eager.
```

A comma after the conditional cause which comes before the main clause helps reading.

I think “away” is correct when you're referring to an upcoming release; whereas “out” would imply an already released version six months ago.

*“…to squeeze it in in the last minute”* could be hard to read because of two adjacent ‘in’s.